src/views/Home/LandingPage.js

```diff
@@ -0,0 +1,104 @@
+import React, { useState } from "react";
+import { useNavigate } from "react-router-dom";
+import "../../sass/landing.scss";
+import parisImage from "../../images/landing-destinations-paris.jpg";
+import tokyoImage from "../../images/landing-destinations-tokyo.jpg";
+import newYorkImage from "../../images/landing-destinations-newyork.jpg";
+import heroImage from "../../images/landing-hero-background.jpg";
+
+// Destination data array (AI Generated descriptions)
+const destinationsData = [
+  {
+    name: "Paris",
+    imagePath: parisImage,
+    credit: 'Image by <a href="https://pixabay.com/users/dnovac-485744/?utm_source=link-attribution&utm_medium=referral&utm_campaign=image&utm_content=4011964">Dan Novac</a> from <a href="https://pixabay.com//?utm_source=link-attribution&utm_medium=referral&utm_campaign=image&utm_content=4011964">Pixabay</a>',
+    description: "Experience the romance and charm of the City of Light. From the iconic Eiffel Tower to the artistic Louvre, Paris offers a perfect blend of history, culture, and cuisine."
+  },
+  {
+    name: "Tokyo",
+    imagePath: tokyoImage,
+    credit: 'Image by <a href="https://pixabay.com/users/pharaoh_ezypt-3065661/?utm_source=link-attribution&utm_medium=referral&utm_campaign=image&utm_content=5664836">Pharaoh_EZYPT</a> from <a href="https://pixabay.com//?utm_source=link-attribution&utm_medium=referral&utm_campaign=image&utm_content=5664836">Pixabay</a>',
+    description: "Dive into the futuristic metropolis of Tokyo. A city where ancient traditions meet cutting-edge technology, offering visitors a unique blend of cultural experiences and modern attractions."
+  },
+  {
+    name: "New York",
+    imagePath: newYorkImage,
+    credit: 'Image by <a href="https://pixabay.com/users/wiggijo-3628174/?utm_source=link-attribution&utm_medium=referral&utm_campaign=image&utm_content=1778564">Jo Wiggijo</a> from <a href="https://pixabay.com//?utm_source=link-attribution&utm_medium=referral&utm_campaign=image&utm_content=1778564">Pixabay</a>',
+    description: "Explore the Big Apple, a melting pot of cultures and dreams. From Broadway shows to Central Park, New York City offers endless entertainment and iconic landmarks at every turn."
+  }
+];
+
+// Reusable component for destination cards
+const DestinationCard = ({ destination }) => (
+  <div className="landing-page__destination-card">
+    <div className="landing-page__destination-image">
+      <img src={destination.imagePath} alt={destination.name} />
+      <small className="image-credit" dangerouslySetInnerHTML={{ __html: destination.credit }}></small>
```

Comment:
What output does destination.credit give? is it html? I know that we control the source of the information however if theres a way to avoid using dangerouslySetInnerHTML then I probably would.  

Again haven't seen the small tag used before.  Another nice bit of learning for me.